docs/servers/add-digitalocean-server.md

```diff
@@ -11,16 +11,16 @@ links:
     featured:
 ---
 
-> If you don't have an DigitalOcean credential, please follow the guide {% mentionPost "/docs/users/credentials/add-digitalocean-credential" %}
+> If you don't have an DigitalOcean credential, please follow the guide {% mentionPost "/docs/credentials/add-digitalocean-credential" %}
 
 1. On Devopness, navigate to a project then select an environment
 1. If there's no other server in the selected environment, just click `ADD SERVER`
 1. If a list of resource cards is displayed, find the `Servers` card and click View on it
 1. On the upper-right corner of the list click `ADD SERVER`
 1. Choose `DigitalOcean` as cloud provider
-1. Select an DigitalOcean credential using the drop-down menu
-    > If no credential is listed or you want to use a different one, follow the guide {% mentionPost "/docs/users/credentials/add-credential" %}
-1. Click `NEXT`
+1. With the `Cloud Provider` selected, the drop-down `Credentials` list all credentials of this `Cloud Provider`, select a credential 
+    > If no credential is listed or you want to use a different one, click `Create a new credential` and follow the guide {% mentionPost "/docs/credentials/add-credential" %}
+1. Once a `Cloud Provider Credential` is selected, click in `Next` button
 1. Select an available region using the drop-down menu
 1. Select an available instance type using the drop-down menu
 1. Set the `Disk size`
```

Comment:
The same diff can be applied to aws and gcp articles, to simplify steps: "Follow the steps" and "wait for ... to be completed"

```
1. Follow the prompts to complete the configuration of the new `Server`
1. Review server information and settings. If everything is correct, click `CONFIRM`
    > Use the `PREVIOUS` button if you need to modify any server details
1. Wait for the `server:provision` action to be completed
```

Reply:
> to simplify steps

The steps mentioned here are the ones below

https://github.com/devopness/devopness/blob/22ca6153170faac8bc68183bbb0afd247f230feb/docs/servers/add-digitalocean-server.md?plain=1#L23-L32

Reply:
Done, can you review again?